test/turbulence_closures.jl

```diff
@@ -0,0 +1,26 @@
+using Breeze
+using GPUArraysCore: @allowscalar
+using Oceananigans
+using Test
+
+const TC = Breeze.TurbulenceClosures
+
+@testset "Time stepping with TurbulenceClosures [$(FT)]" for FT in (Float32, Float64)
+    Oceananigans.defaults.FloatType = FT
+    grid = RectilinearGrid(default_arch; size=(8, 8, 8), x=(0, 100), y=(0, 100), z=(0, 100))
+
+    # Try two closures from Oceananigans
+    for closure in (SmagorinskyLilly(), AnisotropicMinimumDissipation())
+        @testset let closure=closure
+            model = AtmosphereModel(grid; closure=closure)
+            @test try
```

Comment:
Why may this fail? Don't we want to see the error?

Reply:
whats a good pattern? shoudl we just atke the time-step and then write `@test true` after? basically i want to test that it doesn't throw an error, is all